### PR TITLE
update repos dependancies all packaged minus jest typescript related deps

### PR DESCRIPTION
**Pull request by zainen**
build correctly all test suits pass, integration tests have lots of timeout tests to be confident nothing was missed. 

please double check package bumps 😓 

breaking changes reasons

Husky -> project non-used breaking changes 
```
Environment variables:

HUSKY_SKIP_HOOKS is replaced by HUSKY.
HUSKY_SKIP_INSTALL is replaced by HUSKY.
HUSKY_GIT_PARAMS is removed. Instead Git parameters should be used directly in scripts (e.g. $1).
PATH for locally installed tools is not automatically set anymore. You'll need to use your package manager to run them.
```
Lerna -> minimum node version >= 14
Lint-staged -> minimum node >= 14.13.1
Gh-pages -> minimum node > 10

- [ ] Your code builds cleanly without any errors or warnings
- [ ] You have added unit tests
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

## Release Note Draft Snippet

__If relevant, please write a summary of your change that will be suitable for
inclusion in the Release Notes for the next Taquito release.__

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "example:scan-path-ledger": "node -r ts-node/register --preserve-symlinks scan-path-ledger.ts"
   },
   "dependencies": {
-    "@ledgerhq/hw-transport-node-hid": "^6.20.0",
+    "@ledgerhq/hw-transport-node-hid": "^6.27.1",
```
Comment (dsawali) [inline example/package.json:43]: are the examples running without errors as well?

File: integration-tests/package-lock.json [hunk 2]
```diff
@@ -1,9150 +1,8 @@
 {
 	"name": "integration-tests",
 	"version": "13.0.0",
-	"lockfileVersion": 2,
+	"lockfileVersion": 1,
```
Comment (dsawali) [inline integration-tests/package-lock.json:4]: what npm version are you using? I think we'd like to make sure the lockfileVersions are consistently `2`

File: packages/taquito-beacon-wallet/package-lock.json [hunk 3]
```diff
@@ -1,8922 +1,8 @@
 {
 	"name": "@taquito/beacon-wallet",
 	"version": "13.0.0",
-	"lockfileVersion": 2,
+	"lockfileVersion": 1,
```
Comment (dsawali) [inline packages/taquito-beacon-wallet/package-lock.json:4]: same here, might want to double check all the other package-locks as well

File: packages/taquito/package.json [hunk 4]
```diff
@@ -87,36 +87,36 @@
...
     "buffer": "^6.0.3",
     "colors": "^1.4.0",
     "coveralls": "^3.1.1",
     "cross-env": "^7.0.3",
-    "eslint": "^8.5.0",
+    "eslint": "^8.17.0",
     "jest": "^26.6.3",
     "jest-config": "^26.6.3",
-    "lint-staged": "^12.1.2",
+    "lint-staged": "^13.0.1",
```
Comment (dsawali) [inline packages/taquito/package.json:104]: could you confirm whether the pre commit hook was still running when you did a `git commit`?

**Review by dsawali — CHANGES_REQUESTED** [review]

File: integration-tests/package-lock.json (hunk 2 above)
Comment (zainen) [reply integration-tests/package-lock.json:4]: running 6.14.17

File: example/package.json (hunk 1 above)
Comment (zainen) [reply example/package.json:43]: i couldnt see any problems but i can go throught them one at a time

File: integration-tests/package-lock.json (hunk 2 above)
Comment (zainen) [reply integration-tests/package-lock.json:4]: which npm version should i be running?

Comment (zainen) [reply integration-tests/package-lock.json:4]: i have to run node 14 to make lerna commands happy

File: packages/taquito/package.json (hunk 4 above)
Comment (zainen) [reply packages/taquito/package.json]: I havent noticed anything different from my usual but i'll make a dummy branch to be sure

File: integration-tests/package-lock.json (hunk 2 above)
Comment (dsawali) [reply integration-tests/package-lock.json:4]: Roxane and I both run npm version 8.1.0

Comment (dsawali) [reply integration-tests/package-lock.json:4]: the lock files should auto adjust to version 2 if you use version 8 and above I believe

**Pull request closed without merging**